x-pack/plugins/endpoint/server/routes/endpoints.ts

```diff
@@ -5,19 +5,44 @@
  */
 
 import { IRouter } from 'kibana/server';
-import { endpoints2 } from './endpoints_stubs';
+import { schema } from '@kbn/config-schema';
+
+export function registerEndpointsApi(router: IRouter) {
+  const endpointsHandler = async (context, req, res) => {
+    try {
+      if (!context.endpointPlugin) return res.internalError({ body: 'endpointPlugin is disabled' });
+      if ('id' in req.params) {
```

Comment:
Why the duplicate usage of the same handler? Seems kind of awkward that you can use the same endpoints for the same purpose  when one is REST-fully scoped to a given endpoint.